docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -166,6 +166,17 @@ For native app tests on real devices, you must provide a location from which you
 "appium:app","https://github.com/test-apps/ios-app.ipa"
 ```
 
+You can also install an app during a test by using `mobile: installApp` command. 
```

Comment:
Maybe change it to this


You can also install a dependent app or an app upgrade during a test by using [](http://appium.io/docs/en/commands/device/app/install-app/) command. 

```java
driver.installApp("https://github.com/saucelabs/my-demo-app-rn/releases/download/v1.3.0/Android-MyDemoAppRN.apk");
```